std/container/slist.d

```diff
@@ -123,6 +123,19 @@ struct SList(T)
         return n;
     }
 
+    private static Node * findNodeByValue(Node * n, T value)
+    {
+        assert(n);
```

Comment:
Needs an error message, e.g. "The node can't be null"

(I know that there are other methods in this module don't do this either, but that's no reason to continue using the old style)